### PR TITLE
Keyword `description`

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
This PR addresses #72.

### CODE REVIEW

File: content/2020-12/meta-data/description.markdown [hunk 1]
```diff
@@ -27,3 +27,141 @@ Annotations
 -----------
 
 This keyword produces the description as the annotation value.
+
+## Explanation
+
+The `description` keyword in JSON Schema is used to provide additional explanatory information about the schema or its parts. It does not affect data validation but serves as an informative annotation. The value of this keyword must be a string.
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:33]:
```suggestion
The `description` keyword in JSON Schema is used to provide a human readable description for the schema. It does not affect data validation but serves as an informative annotation. The value of this keyword must be a string.
```

I guess a schema implies its parts?

File: content/2020-12/meta-data/description.markdown [hunk 2]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+
+## Examples
+
+{{<schema `Schema with 'description' keyword`>}}
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "The age of a person must not be less than zero",
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:40]: I'm not sure this example is very good. The intention of the `description` keyword is not to express a constraint in prose, but to document the schema. Maybe a description of "The age of a person" makes more sense?

File: content/2020-12/meta-data/description.markdown [hunk 3]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "Personal information of a user",
+  "properties": {
+    "name": { "type": "string" },
+    "age": { "type": "number" }
+  },
+  "if": {
+    "description": "if block",
+    "properties": {
+      "age": { "description": "'if' true", "minimum": 18 }
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:69]: You are adding single quotes around `if` here, but not doing the same for `else` and `then`. Maybe pick one way and stick to it?

Comment (jviotti) [reply content/2020-12/meta-data/description.markdown:69]: Also, this description is similar to the outer description in `if` and same for then/else. Maybe make this specific annotation something like "age"?

File: content/2020-12/meta-data/description.markdown [hunk 4]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+    "valid": true,
+    "keywordLocation": "/then/properties/eligible/description",
+    "instanceLocation": "/eligible",
+    "annotation": "then applied"
+  },
+  // ...
+]
+{{</instance-annotation>}}
+
+{{<schema `Schema with multiple annotations for the same instance`>}}
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:132]: Cool example!

File: content/2020-12/meta-data/description.markdown (hunk 3 above)
Comment (AgniveshChaubey) [reply content/2020-12/meta-data/description.markdown:69]: Sure!

File: content/2020-12/meta-data/description.markdown [hunk 5]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+    "valid": true,
+    "keywordLocation": "/if/description",
+    "instanceLocation": "",
+    "annotation": "if block"
+  },
+  {
+    "valid": true,
+    "keywordLocation": "/if/properties/age/description",
+    "instanceLocation": "/age",
+    "annotation": "'if' true"
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:114]: I think the annotations on this example are now out of the date?

File: content/2020-12/meta-data/description.markdown [hunk 6]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+{{<instance-pass `An instance with a numeric value is valid`>}}
+45
+{{</instance-pass>}}
+
+{{<instance-annotation>}}
+{
+  "valid": true,
+  "keywordLocation": "/description",
+  "instanceLocation": "",
+  "annotation": "The age of a person must not be less than zero"
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:54]: This annotation is outdated too

File: content/2020-12/meta-data/description.markdown [hunk 7]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+    "annotation": "then applied"
+  },
+  // ...
+]
+{{</instance-annotation>}}
+
+{{<schema `Schema with multiple annotations for the same instance`>}}
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "Name of the person should not include special characters",
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:135]: Like on my other comment before, let's not describe constraints on `description`. Just a statement of the instance. Like "A person name"

File: content/2020-12/meta-data/description.markdown (hunk 6 above)
Comment (AgniveshChaubey) [reply content/2020-12/meta-data/description.markdown:54]: Ahh!! Yesterday was quite stressful for me, and I unintentionally became a bit less careful when addressing the suggestions. I apologize for that. I need to be more mindful of your time. I'll make sure this doesn't happen again.

File: content/2020-12/meta-data/description.markdown [hunk 8]
```diff
@@ -27,3 +27,141 @@ Annotations
...
+    "valid": true,
+    "keywordLocation": "/if/description",
+    "instanceLocation": "",
+    "annotation": "if block"
+  },
+  {
+    "valid": true,
+    "keywordLocation": "/if/properties/age/description",
+    "instanceLocation": "/age",
+    "annotation": "'Age"
```
Comment (jviotti) [inline content/2020-12/meta-data/description.markdown:114]:
```suggestion
    "annotation": "Age"
```

There is a weird comma here

File: content/2020-12/meta-data/description.markdown (hunk 6 above)
Comment (jviotti) [reply content/2020-12/meta-data/description.markdown]: No worries! :)

**Review by jviotti — APPROVED** [review]
Just one comment about an extraneous character, but looks good otherwise

**Pull request merged**